packages/@aws-cdk/aws-events-targets/test/logs/integ.log-group.ts

```diff
@@ -51,5 +54,36 @@ timer3.addTarget(new targets.CloudWatchLogGroup(importedLogGroup, {
   retryAttempts: 2,
 }));
 
+const integ = new IntegTest(app, 'LogGroup', {
+  testCases: [stack],
+});
+
+const putEventsDate = Date.now().toString();
+const expectedValue = `abc${putEventsDate}`;
+
+const putEvent = integ.assertions.awsApiCall('EventBridge', 'putEvents', {
+  Entries: [
+    {
+      Detail: JSON.stringify({
+        date: expectedValue,
+      }),
+      DetailType: 'cdk-integ-custom-rule',
+      Source: 'cdk-integ',
+    },
+  ],
+});
+const assertionProvider = putEvent.node.tryFindChild('SdkProvider') as AssertionsProvider;
```

Comment:
Tangential question: do users have to do this for every assertion?

Reply:
No, and i've added an item to our list to expose this ability on the `AssertionsProvider`. It's supposed to figure out the permissions based on the API call, but I keep running into cases where it can't figure it out.